src/core/qgsmaphittest.h

```diff
@@ -71,6 +72,13 @@ class CORE_EXPORT QgsMapHitTest
      */
     bool legendKeyVisible( const QString &ruleKey, QgsVectorLayer *layer ) const;
 
+    /**
+     * Tests whether a given legend key is visible for a specified layer.
```

Comment:
This implies that raster classes are also handled just like for the vector layers. 
```suggestion
     * Tests whether a raster layer is visible.
```